test/api/pack/hierarchical_binning_test.cpp

```diff
@@ -79,6 +79,35 @@ TEST(hierarchical_binning_test, another_example)
     EXPECT_EQ(header_buffer.str() + output_buffer.str(), expected_file);
 }
 
+TEST(hierarchical_binning_test, high_level_max_bin_id_is_0)
```

Comment:
Shouldn't this one be called `only_top_level` or something? Or is it important that the `max_bin_id` is exactly `0`

Reply:
Yes. It is the edge case that the trace it at position 0, the very first bin, and that is a split bin (rather not-merged-bin because it is a single bin, I used to call them split bins anyway) and that has the highest weight.

Reply:
It's ok, but add what you just wrote as comment to the test?

Reply:
sure